_posts/2014-07-29-installing.md

```diff
@@ -329,7 +340,7 @@ Change file permissions so that we can update the rules and change group and rul
  $ chmod g+rw /etc/MailScanner/rules/*.rules
 ```
 
-See also the INSTALL docs in `tools/MailScanner_rule_editor` and `tools/Cron_jobs` directories.
+See also the INSTALL docs in `tools/MailScanner_rule_editor/INSTALL` and in `tools/Cron_jobs/INSTALL` directories.
```

Comment:
misleading phrase: is `tools/MailScanner_rule_editor/INSTALL` a directory? no it isn't but seems like it is.
Fow now revert to old version; before 1.2.0 all `INSTALL` files need to be converted to online documentation and removed from MailWatch repository